### PR TITLE
add detail for the api POST /api/crypto/withdraw

### PR DESCRIPTION
**Pull request by ponlamai-sangapong**
- add a description for 'net' input param
- add  Additional section

### CODE REVIEW

File: restful-api.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 # RESTful API for Bitkub (2022-06-02)
-
 # Releases
+* 2022-06-21 Add field ```net``` for the api [POST /api/crypto/withdraw](#post-apicryptowithdraw)
```
Comment (iamgoangle) [inline restful-api.md:4]: Should be `Added` field...?

File: restful-api.md [hunk 2]
```diff
@@ -1192,6 +1199,14 @@ Check trading credit balance.
 }
 ```
 
+# Additional
+For the  use of `cur`(currency) for any APIs request. Please be cautious of these cryptocurrency when you specified on the request.
+
+Name|Currency
+---|---
+Terra Classic|`LUNA`
+Terra 2.0|`LINA2`
```
Comment (iamgoangle) [inline restful-api.md:1208]: `LINA` I think is a hero in dota. Please revisit this again.

Comment (ponlamai-sangapong) [reply restful-api.md:1208]: my bad

**Review by iamgoangle — APPROVED** [review]
LGTM

**Pull request merged**